src/lib/autocomplete/BUILD.bazel

```diff
@@ -42,3 +42,30 @@ sass_library(
   name = "theme",
   srcs = glob(["**/*-theme.scss"]),
 )
+
+ng_test_library(
+  name = "autocomplete_test_sources",
+  srcs = glob(["**/*.spec.ts"]),
+  deps = [
+    "@angular//packages/forms",
+    "@angular//packages/platform-browser",
+    "@angular//packages/platform-browser/animations",
+    "@rxjs",
+    "@rxjs//operators",
+    "//src/cdk/bidi",
+    "//src/cdk/keycodes",
+    "//src/cdk/overlay",
+    "//src/cdk/scrolling",
+    "//src/cdk/testing",
+    "//src/lib/core",
+    "//src/lib/form-field",
+    "//src/lib/input",
+    ":autocomplete"
+  ]
+)
+
+ng_web_test_suite(
+  name = "unit_tests",
+  deps = [":autocomplete_test_sources"],
+  prebuilt_theme = True,
```

Comment:
Why add an attribute for this rather than always include it?

Reply:
Only a few tests that measure pixels and computed styles need the prebuilt theme to be available within Karma.

For the other tests, it would just cause an _unnecessary_ compilation of the prebuilt theme that takes some time.

Reply:
Done. I had had to rerun all tests locally because GitHub web hooks are currently not working.

I've addressed your feedback. Note that I still kept it explicit through a named parameter (`prebuilt_theme`) because we use the macro for the CDK tests as well.. and having something like `mat_web_test_suite` feels a bit overkill.